packages/app-mobile/components/screens/ConfigScreen.tsx

```diff
@@ -373,12 +374,28 @@ class ConfigScreenComponent extends BaseScreenComponent {
 		}
 
 		BackButtonService.addHandler(this.handleBackButtonPress);
+		await this.loadInboxEmail();
 	}
 
 	public componentWillUnmount() {
 		BackButtonService.removeHandler(this.handleBackButtonPress);
 	}
 
+	private async loadInboxEmail() {
+		if (this.props.settings['emailToNote.inboxEmail'] !== '') return;
+		if (this.props.settings['sync.target'] !== SyncTargetRegistry.nameToId('joplinCloud')) return;
+
+		const syncTarget = reg.syncTarget();
+		const fileApi = await syncTarget.fileApi();
+		const api = fileApi.driver().api();
+
+		const owner = await api.exec('GET', `api/users/${api.userId}`);
+
+		if (owner.inbox_email) {
+			Setting.setValue('emailToNote.inboxEmail', owner.inbox_email);
+		}
+	}
```

Comment:
Same here - you should use your emailFetcher code from the other pull request. That's the benefit of having a shared codebase, we can (and should) reuse code between desktop and mobile app